src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="center-block">
+    <div class="calendar calendar-first-day-sun calendar--month-length-31">
```

Comment:
Check [BEM](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem) rules
![image](https://github.com/mate-academy/layout_calendar/assets/91411692/b5c799c5-ddf6-43fe-806c-61a9b5052745)

Reply:
Can you show me an example? Because i am not sure that i understood correctly.